### PR TITLE
Setting location radius

### PR DESCRIPTION
**Pull request by nikita206**
**Context**
Users can use a slider to adjust radius of miles that they want to see gym buddies in. Displays users according to the radius by calculating coordinates and the fitness level of the user. Displays a blank page as of now when there are no users in that area (will update the blank page).
**Code changes**
1. Added code to store latitudes and longitudes of the location set by user
2. Implemented query to perform the calculation of coordinates within radius
3. Implemented a slider for users to adjust the radius
4. Added if-else conditions to ensure no error is thrown when the array of posts is null
**Testing**
https://user-images.githubusercontent.com/56750668/182960642-6ef26d7d-9bf4-4717-9922-00c614c2548b.mp4

### CODE REVIEW

File: StayFit/Tab Bar Controller/Community View/CommunityViewController.m [hunk 1]
```diff
@@ -128,6 +135,7 @@ -(void) fetchRecipesPosts{
         }
         else {
             //throws an error with descriptions if no posts are found in the array
+
```
Comment (raghavan-rahul) [inline StayFit/Tab Bar Controller/Community View/CommunityViewController.m]: remove extra spaces

File: StayFit/Tab Bar Controller/Community View/CommunityViewController.m [hunk 2]
```diff
@@ -55,14 +55,19 @@ - (IBAction)segact:(id)sender {
...
     //stores all the posts in an array
     self.arrayOfFitnessPosts = [[NSMutableArray alloc] init];
     [self fetchFitnessPosts];
 }
 
 -(void) recipesCase{
-    //stores all the posts in an array
     self.addButton.hidden = false;
+    //hides the slider in recipes segment
+    self.slider.hidden = true;
```
Comment (raghavan-rahul) [inline StayFit/Tab Bar Controller/Community View/CommunityViewController.m:67]: i thin the comment is not necessary over here

**Comment by raghavan-rahul** [comment]
Can you add a Test plan as well for example a screenshot of the code changes

**Comment by nikita206** [comment]
> Collaborator

yes, I added it on the workplace. Should I add it on the PR as well?

**Review by raghavan-rahul — APPROVED** [review]

**Pull request merged**